src/formatter/formatComplexDataStructure.spec.js

```diff
@@ -86,4 +86,20 @@ describe('formatComplexDataStructure', () => {
   it('should format an empty array', () => {
     expect(formatComplexDataStructure([], false, 0, options)).toEqual('[]');
   });
+
+  it('should format an object that contains a date', () => {
```

Comment:
I guess theses tests are more related to `sortObject` beavior than the `formatComplexDataStructure` one. We should move then in the `sortObject.spec.js` file

Reply:
`sortObject` has its own test for that. I think left it here since not only `sortObject` is used to process objects and other logic can break things too.